### ISSUE TITLE
updates?

### ISSUE THREAD

**Issue by AAvalos82** [CoreArray/JSeqArray.jl#1]
Any thoughts on revisiting this port? I have been curious about the implementation of these systems on Julia but it seems this effort has "died on the vine". I was looking forward to a version of these tools available for Julia, and indeed their absence in that language has been one of the main reasons why I have stuck with R.

**Comment by zhengxwen** [comment]
I do have a plan to relaunch the GDS implementation on Julia late 2019 or early 2020. But I don't have a strong motivation to "duplicate" R packages (e.g., SNPRelate, GENESIS, GMMAT) on Julia for statistical association tests.
Instead, I might focus on large-scale machine learning using genomic data. Julia is one of the ideal programming languages for high-performance large-scale machine learning.

**Comment by AAvalos82** [comment]
Fair enough. Looking forward to it though, it would be nice to have the GDS backbone implemented at least.
